### PR TITLE
Lossless 652: Staker contract and tests fixes

### PR DESCRIPTION
**Pull request by Ignacio-Freire**

### CODE REVIEW

File: test/contract based tests/EnvironmentTests.js [hunk 1]
```diff
@@ -87,6 +87,26 @@ describe('Lossless Environment', () => {
...
+          ).to.be.revertedWith('LSS: Must be admin');
+        });
+
+        it('should revert when setting up as zero address', async () => {
+          await expect(
+            env.lssStaking.connect(adr.lssAdmin).setStakingToken(ZERO_ADDRESS),
+          ).to.be.revertedWith('LERC20: Cannot be zero address');
+        });
+
+        it('should not revert when not admin', async () => {
```
Comment (DomantasP) [inline test/contract based tests/EnvironmentTests.js:125]: should not revert when admin

File: test/contract based tests/EnvironmentTests.js [hunk 2]
```diff
@@ -88,6 +89,26 @@ describe('Lossless Environment', () => {
...
+          ).to.be.revertedWith('LSS: Must be admin');
+        });
+
+        it('should revert when setting up as zero address', async () => {
+          await expect(
+            env.lssStaking.connect(adr.lssAdmin).setLosslessGovernance(ZERO_ADDRESS),
+          ).to.be.revertedWith('LERC20: Cannot be zero address');
+        });
+
+        it('should not revert when not admin', async () => {
```
Comment (DomantasP) [inline test/contract based tests/EnvironmentTests.js:105]: when admin

File: test/domain based tests/setCommunityStakeAmount.js [hunk 3]
```diff
@@ -38,5 +38,11 @@ describe(scriptName, () => {
         await env.lssStaking.stakingAmount(),
       ).to.be.equal(3);
     });
+
+    it('should emit StakingAmountChanged event', async () => {
+      await expect(
+        env.lssStaking.connect(adr.lssAdmin).setStakingAmount(3),
+      ).to.emit(env.lssStaking, 'StakingAmountChanged').withArgs(3);
```
Comment (DomantasP) [inline test/domain based tests/setCommunityStakeAmount.js:45]: Can we check args?

File: test/contract based tests/EnvironmentTests.js (hunk 1 above)
Comment (Ignacio-Freire) [reply test/contract based tests/EnvironmentTests.js:125]: Addressed on 4ababf0

File: test/contract based tests/EnvironmentTests.js (hunk 2 above)
Comment (Ignacio-Freire) [reply test/contract based tests/EnvironmentTests.js]: Addressed on 4ababf0

File: test/domain based tests/setCommunityStakeAmount.js (hunk 3 above)
Comment (Ignacio-Freire) [reply test/domain based tests/setCommunityStakeAmount.js:45]: This one is checking for the literal value 3 as the event is defined as it follows: 

`event StakingAmountChanged(uint256 indexed newAmount);`

**Pull request merged**